app/models/manifest.rb

```diff
@@ -56,7 +58,7 @@ def self.rebuild_index
   end
 
   def self.authorized_search(params, current_user=nil)
```

Comment:
I might call this `user` rather than `current_user` since we're in a model and models don't have access to the `current_user` var like a controller or view might